packages/@sanity/portable-text-editor/src/editor/plugins/createWithSchemaTypes.ts

```diff
@@ -59,6 +60,13 @@ export function createWithSchemaTypes({
         const key = span._key || keyGenerator()
         Transforms.setNodes(editor, {...span, _type: schemaTypes.span.name, _key: key}, {at: path})
       }
+
+      // catches cases when the children are missing keys but excludes it when the normalize is running the node as the editor object
+      if (node._key === undefined && !_.isEqual(node, editor)) {
```

Comment:
@skogsmaskin had to add the condition since the tests showed that the editor itself was falling in this condition (since it has no _key itself)